Assets/MixedRealityToolkit-Examples/UX/Scripts/Controls/SliderGestureControl.cs

```diff
@@ -24,7 +24,7 @@ public class SliderGestureControl : GestureInteractiveControl
         public GameObject CenteredDot;
 
         [Tooltip("Sends slider event information on Update")]
-        public UnityEvent OnUpdateEvent;
+        public UnityEventFloat OnUpdateEvent;
```

Comment:
Why are we even using a unity event here anyway?

Reply:
I believe events are being used so that one or many callback methods can be specified from within the editor.

Reply:
yeah that's exactly the problem. It's better to do it in code.

Reply:
I do not agree.  If a user wants to use the slider prefab and drags it onto the scene, then the current implementation allows for them to wire up some functionality without writing any code.

This is also true for the current implementation of the buttons, checkbox and other UX prefabs provided by the holotoolkit-examples package

Reply:
It's not any different than having a float field and a game object field and referencing it that way.

Believe me, it's a lot easier to deal with.

Reply:
> This is also true for the current implementation of the buttons, checkbox and other UX prefabs provided by the holotoolkit-examples package.

Yeah, those are also doing it wrong.

Reply:
I think I understand, you could just have a reference to a game object instead and have the code change that value directly on update.

The current implementation of the controls using the event system does however allow for other things, such as triggering multiple methods on multiple objects.

Reply:
I think this discussion is addressing a larger issue, namely "should the toolkit controls use unity events or some other system".

This PR is addressing a minor issue in that the update event of the slider does not allow the updated value to be sent.  

Would it be pragmatic to add this update to the current system?  And discuss a refactor to the controls event system in another issue?  I believe that there will be arguments for both sides of this discussion.